my_list.rb

```diff
@@ -0,0 +1,31 @@
+$LOAD_PATH << '.'
+require 'my_enumerable'
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you use  `require_relative` to import `MyEnumerable` since using `require_relative` for files in your directory instead of `require` is a ruby convention. 
Require should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory.

Reply:
@tresorsawasawa thanks for your feedback